src/execution_plan/ops/op_argument_list.h

```diff
@@ -20,7 +20,8 @@ typedef struct {
 
 OpBase *NewArgumentListOp
 (
-	const ExecutionPlan *plan
+	const ExecutionPlan *plan,
+	const char **variables
```

Comment:
Consider renaming to `bound_vars`
would be nice to add a bit of comments / documentation

Reply:
I took the same code from op_argument